src/components/image/Image.component.tsx

```diff
@@ -1,6 +1,6 @@
 import React from 'react';
 
-import portrait from '@/assets/portrait.png';
+import portrait from '@/assets/portrait.svg';
```

Comment:
what the reason to create Image component. You can just import image from assets into component and use, without extra components.